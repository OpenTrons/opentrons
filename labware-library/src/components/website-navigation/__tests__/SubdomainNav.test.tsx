// tests for top navbar
import * as React from 'react'
import { shallow } from 'enzyme'

import { SubdomainNav, SUBDOMAIN_NAV_LINKS } from '..'

describe('SecondaryNav', () => {
  it('component renders', () => {
    const tree = shallow(<SubdomainNav />)

    expect(tree).toMatchSnapshot()
  })

  it('<li> rendered match link data', () => {
    const tree = shallow(<SubdomainNav />)

    expect(tree.find('li')).toHaveLength(SUBDOMAIN_NAV_LINKS.length)
  })
})
