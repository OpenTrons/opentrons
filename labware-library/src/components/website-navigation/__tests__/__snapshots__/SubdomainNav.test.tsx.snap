// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondaryNav component renders 1`] = `
<ul
  className="subdomain_nav_contents"
>
  <li
    key="0"
  >
    <a
      className="subdomain_link"
      data-gtm-action="click"
      data-gtm-category="l-toolbar"
      data-gtm-label="opentrons-api"
      href="https://docs.opentrons.com/"
      rel="noopener noreferrer"
      target="_blank"
    >
      Python API
    </a>
  </li>
  <li
    key="1"
  >
    <Link
      className="subdomain_link"
      data-gtm-action="click"
      data-gtm-category="l-toolbar"
      data-gtm-label="labware-library"
      to="/"
    >
      Labware Library
    </Link>
  </li>
  <li
    key="2"
  >
    <a
      className="subdomain_link"
      data-gtm-action="click"
      data-gtm-category="l-toolbar"
      data-gtm-label="protocol-library"
      href="https://protocols.opentrons.com/"
      rel="noopener noreferrer"
      target="_blank"
    >
      Protocol Library
    </a>
  </li>
  <li
    key="3"
  >
    <a
      className="subdomain_link"
      data-gtm-action="click"
      data-gtm-category="l-toolbar"
      data-gtm-label="protocol-designer"
      href="https://designer.opentrons.com/"
      rel="noopener noreferrer"
      target="_blank"
    >
      Protocol Designer
    </a>
  </li>
</ul>
`;
